src/devices/CharacterLcd/ICharacterLcd.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Iot.Device.CharacterLcd
+{
+    /// <summary>
+    /// Interface for character LCD Low-Level handler
+    /// </summary>
+    public interface ICharacterLcd : IDisposable
+    {
+        /// <summary>
+        /// Enable/disable the backlight. (Will always return false if no backlight pin was provided.)
+        /// </summary>
+        bool BacklightOn { get; set; }
```

Comment:
NIT: for naming, should we name these EnableBacklight and EnableDisplay?

Reply:
See above https://github.com/dotnet/iot/pull/859#discussion_r381024735. Should we do a breaking change for this, or just keep them both in the current implementation?

Reply:
I see, didn't realize it would be breaking, I'm fine with keeping as is.